content/en/docs/reference/glossary/jwt.md

```diff
@@ -0,0 +1,19 @@
+---
+title: JSON Web Token (JWT)
+id: jwt
+date: 2022-12-06
+full_link: https://www.rfc-editor.org/rfc/rfc7519
+short_description: >
+  A digitally signed JSON object that can be used for authentication.
```

Comment:
```suggestion
  An authentication token format based on JSON data.
```

Reply:
Done

Reply:
This suggestion is pending.

Reply:
Weird, thought I committed it. done now.